doc/whats_new/v0.22.rst

```diff
@@ -90,7 +95,7 @@ Changelog
   remove constant features due to numerical instability, by using range
   rather than variance in this case.
   :pr:`13704` by `Roddy MacSween <rlms>`.
-  
+
```

Comment:
please revert (above too)

Reply:
Sorry for that, trailing whitespaces get automatically removed.